.gitignore

```diff
@@ -8,6 +8,7 @@ logs
 
 # virtualenv
 .Python
+.python-version
```

Comment:
@konklone I installed pyenv and it added that file, I assumed we didn't need that on github so I added to .gitignore